### ISSUE TITLE
Ignore dependencies coming from webpack-dev-server

### ISSUE THREAD

**Issue by sandersky** [darrenscerri/duplicate-package-checker-webpack-plugin#11]
This plugin currently informs me of duplicates where one of the instances comes from `webpack-dev-server`. Ideally it'd be nice if this plugin simply ignores files from that module to reduce noise, or potentially even provides an optional function for consumers to identify whatever they deem a false positive. Maybe something like:

```js
new DuplicatePackageCheckerPlugin({
  emitError: false,
  isFalsePositive: function () {
    // add custom logic here that returns true when we deem this is a false positive.
    // this will require useful information via argument(s) for custom logic
    return false;
  },
  verbose: true,
})
```

**Comment by darrenscerri** [comment]
Thanks for opening this issue @sandersky. What kind of arguments do you have in mind? I'm currently thinking about adding the package name and full path of the module. Is there something else that you think might be useful?

**Comment by sandersky** [comment]
Glancing at the code real quick what about `instances` and `name` which come from [here](https://github.com/darrenscerri/duplicate-package-checker-webpack-plugin/blob/master/src/index.js#L114)? Your logic could just do an early return in the for loop whenever the custom method returns true to avoid logging the issue or pushing an error to the array.
